<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AdminAnchorageDetails</class>
 <widget class="QDialog" name="AdminAnchorageDetails">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>430</width>
    <height>210</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget{
background-color:rgb(159, 208, 213)
}

QPushButton {
color: rgb(0, 0, 0);
background-color: rgb(137,198,205);
border-color: blue;
border-width: 0px;
border-style: solid;
border-radius: 7;
}

QPushButton:pressed{
color: rgb(0, 0, 0);
background-color: rgb(255, 255, 241);
}

QLabel{
color: rgb(0, 0, 0);
background-color: rgb(153, 153, 153);
selection-color: white;
selection-background-color: rgb(191, 31, 127);
border: 2px;
border-radius: 10px;
padding: 2px 4px;
}

QLineEdit {
color: rgb(127, 0, 63);
background-color: rgb(255, 255, 241);
selection-color: white;
selection-background-color: rgb(191, 31, 127);
border: 2px;
border-radius: 10px;
padding: 2px 4px;
}</string>
  </property>
  <widget class="QLabel" name="labelHead">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>410</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Szczegóły kotwicowiska</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelCapacity">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>200</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Ilość miejsc</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelMaxDraft">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>200</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Maksymalne zanurzenie</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="LabelCostPerHour">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>200</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Koszt na godzinę</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditCapacity">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>50</y>
     <width>200</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditMaxDraft">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>90</y>
     <width>200</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditCostPerHour">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>130</y>
     <width>200</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>170</y>
     <width>200</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Transferuj zmiany</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
